test/run-k8s-integration.sh

```diff
@@ -25,9 +25,12 @@ readonly gke_release_channel=${GKE_RELEASE_CHANNEL:-""}
 readonly teardown_driver=${GCE_PD_TEARDOWN_DRIVER:-true}
 readonly gke_node_version=${GKE_NODE_VERSION:-}
 readonly use_kubetest2=${USE_KUBETEST2:-false}
+readonly migration_test=${MIGRATION_TEST:-false}
 
 export GCE_PD_VERBOSITY=9
 
+readonly GCE_PD_TEST_FOCUS="PersistentVolumes\sGCEPD|[V|v]olume\sexpand|\[sig-storage\]\sIn-tree\sVolumes\s\[Driver:\sgcepd\]|allowedTopologies|Pod\sDisks|PersistentVolumes\sDefault"
```

Comment:
Do you know why we have the restricted focus for migration tests?

Reply:
David wrote this focus a while ago - are there certain in-tree tests you noticed that are not covered here?

Reply:
I have no idea. I just wondered if it was a copy-paste or if we know positively that it covers all the in-tree tests.

Reply:
It's a copy-n-paste. I should probably learn more about the tests.

Reply:
I went through the test cases, especially those in this testgrid https://k8s-testgrid.appspot.com/sig-storage-kubernetes#gce.